README.md

```diff
@@ -12,30 +12,20 @@ First, in the [single-spa application](https://github.com/CanopyTax/single-spa/b
 
 ```js
 import {platformBrowserDynamic} from '@angular/platform-browser-dynamic';
+import {ApplicationRef} from '@angular/core';
 import singleSpaAngular from 'single-spa-angular';
 import mainModule from './main-module.ts';
 import {Router} from '@angular/router';
 
-const ngLifecycles = singleSpaAngular({
+export default singleSpaAngular({
```

Comment:
I'm working on some things that will end up making `export default` be the right thing to do.